styles.css

```diff
@@ -527,14 +530,18 @@ body button:hover {
   .lang-container ul {
     display: flex;
     list-style: none;
-    padding: 0;
+    padding-bottom: 2.5em;
     flex-wrap: wrap;
     flex-direction: row;
     align-items: center;
     justify-content: center;
     background-color: #ebf0ee;
     margin: 24px 30px 24px 30px;
   }
```

Comment:
- Kindly remove this space between the home page and the work card section  by reducing that padding-bottom  so that it will make your page look like a Figma design :+1: 
- Please check below :grinning: 
## your design
![image](https://user-images.githubusercontent.com/95297251/186791436-e122c378-c8a8-415e-aa35-2c26aefe91a6.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/186791479-8761a0bb-9755-43d4-97c2-dd6891ea6883.png)